packages/mdc-chips/chip/mdc-chip.scss

```diff
@@ -118,10 +118,12 @@
 .mdc-chip__icon--leading {
   transition: opacity $mdc-chip-opacity-animation-duration linear;
   opacity: 1;
+  transition-delay: -50ms; 
```

Comment:
Add a comment here to explain what's this line is for

Reply:
Done!